pkg/operator/config.go

```diff
@@ -215,18 +223,26 @@ func ConfigFromEnvironment() *Config {
 	}
 
 	if webhooksPort, ok := os.LookupEnv("WEBHOOKS_PORT"); ok {
-		v, err := strconv.ParseInt(webhooksPort, 10, 32)
+		v, err := strconv.Atoi(webhooksPort)
 		if err != nil {
-			v = DefaultWebhooksPort
-			klog.Errorf("Error parsing WEBHOOKS_PORT environment variable: %v", err)
+			return nil, fmt.Errorf("error parsing WEBHOOKS_PORT (%q) environment variable: %v", webhooksPort, err)
 		}
 
-		config.WebhooksPort = int(v)
+		config.WebhooksPort = v
 	}
 
 	if webhooksCertDir, ok := os.LookupEnv("WEBHOOKS_CERT_DIR"); ok {
 		config.WebhooksCertDir = webhooksCertDir
 	}
 
-	return config
+	if metricsPort, ok := os.LookupEnv("METRICS_PORT"); ok {
+		v, err := strconv.Atoi(metricsPort)
+		if err != nil {
+			return nil, fmt.Errorf("error parsing METRICS_PORT (%q) environment variable: %v", metricsPort, err)
```

Comment:
Let's raise the bar and add unit tests for the new behaviour, testing for both positive/negative results when setting `METRICS_PORT` and `WEBHOOKS_CERT_DIR`.